### PR TITLE
Add Note about cluster scaling [CN-1195]

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: docs/modules/ROOT/pages/cp-subsystem.adoc [hunk 1]
```diff
@@ -22,4 +22,6 @@ When the `pvc` section is configured in `cpSubsystem`, a PVC is created and moun
 [source,yaml,subs="attributes+"]
 ----
 include::ROOT:example$/hazelcast-cp-subsystem-persistence.yaml[]
-----
\ No newline at end of file
+----
+
+NOTE: The cluster cannot be scaled up or down with the CP subsystem enabled. Although it is allowed to pause the cluster, i.e., scaling it to 0, when starting the cluster, you must scale only to the original cluster size; otherwise, it will fail to start.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/cp-subsystem.adoc:27]:
```suggestion
NOTE: The cluster cannot be scaled up or down if the CP subsystem is enabled. However, you can pause the cluster by scaling it to 0 when starting the cluster. You must only scale to the original cluster size or it fails to start.
```

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/cp-subsystem.adoc:27]: Not convinced by my suggestion, I think it needs something else - could you clarify slightly, please, I am reading: _You can pause the cluster by scaling to zero, but if zero wasn't the original cluster size you cannot restart it_, but that doesn't seem logical.

Is it?

Comment (SeriyBg) [reply docs/modules/ROOT/pages/cp-subsystem.adoc:27]: Well, not quite like this. What I intended to say, is that it is allowed to scale to zero, but from zero, it is only allowed to scale to the original value.
For example, let's say we have a cluster of size 3, and we scale it to 0. Then, it is allowed to scale it back to 3, but it is not allowed to scale it to 5, as the original value was 3. Hope I haven't explained it in a more complicated way 😄

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/cp-subsystem.adoc:27]: Not at all more complicated, nicely done - and I've added the example to the note ;-)

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/cp-subsystem.adoc (hunk 1 above)
Comment (SeriyBg) [reply docs/modules/ROOT/pages/cp-subsystem.adoc:27]: Thanks! 👍

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Review by hasancelik — APPROVED** [review]

**Pull request merged**